examples/keygen/keygen.c

```diff
@@ -152,6 +169,11 @@ int TPM2_Keygen_Example(void* userCtx, int argc, char *argv[])
                      TPMA_OBJECT_sign | TPMA_OBJECT_noDA,
                      TPM_ECC_NIST_P256, TPM_ALG_ECDSA);
         }
+        else if (alg == TPM_ALG_SYMCIPHER) {
+            printf("AES Symmetric template with 256 bits, Counter mode\n");
+            rc = wolfTPM2_GetKeyTemplate_Symmetric(&publicTemplate, keyBits,
+                    TPM_ALG_CTR, YES, YES);
```

Comment:
Many of the TPM's don't support CBC or CTR. You may need to always use CFB for this example.

Reply:
I tested this example vs the simulator, because AFAIK most TPMs do not support symmetric operations. 

Tested on the Pi and my SLB9670 part does not like CBC CTR and CFB for AES key generation. Does your SLB part work with CFB?

Reply:
I would like to have some options for additional symmetric types and AES modes. Also the key size.

Reply:
Extra options were added.